### DISCUSSION TITLE
Problem with Oneline feature

### DISCUSSION THREAD

**Discussion by elteia** [Godiesc/firefox-gx#227] [Q&A]
![image](https://github.com/user-attachments/assets/6dde3e4d-c45b-4412-86dd-fdbab204c808)
i can't click the other tabs without using the shortcut for shifting tabs. and the online icons are messed up. please help me fix

**Comment by Godiesc** [comment]
hi,
Did you download the latest theme [Release](https://github.com/Godiesc/firefox-gx/releases/tag/v.9.8) version and are you using Firefox release v.133?
Don't forget that you should have only 3 icons in left side, you have this information in the info page to help you:

![imagen](https://github.com/user-attachments/assets/4b24dc40-d366-43a7-a801-32bee1af4ed4)
